libs/blocks/accordion/accordion.css

```diff
@@ -158,6 +150,18 @@ html[dir="rtl"] .accordion dt button::before {
   background: var(--color-gray-100);
 }
 
+html[dir="rtl"] .accordion dt button {
+  padding: var(--spacing-s) var(--spacing-xs) var(--spacing-s) var(--spacing-m);
+}
+
+html[dir="rtl"] .accordion dt button::before {
+  right: 0;
+}
+
+.section[class*='-up'] dl.accordion {
+  max-width: unset;
+}
+
```

Comment:
I see this makes the accordion wider when in an up pattern. I didn't see any mention of this change in the ticket. We sure this change is ok? 

Thinking bigger picture. Technically the masonry grid can work with any blocks. `masonry` would need this update too. That would be a separate ticket, but wanted to note it.

Reply:
Good call. I can leave this part out. I just noticed all the accordions in an up weren't using the full space available and still had a gap so I figured it should have been this way from the start. But agree, probs a question first.

Reply:
@rgclayton  - I spoke w/ bacom and consonant and they are in agreement that this should actually stay as is so that the accordion is as wide as possible when in a 'ups' grid. I am going to leave this in for the time being and if we need to address/extend this on other grid offerings we can do so in a future PR. Good thoughts on this. Thanks.